--- a/CMakeLists.txt	2009-11-16 11:35:12.000000000 -0500
+++ b/CMakeLists.txt	2009-11-18 01:26:03.671875000 -0500
@@ -4,6 +4,7 @@
 #
 CMAKE_MINIMUM_REQUIRED(VERSION 2.6.1)
 SET(CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/build/cmake")
+INCLUDE(dist.cmake) # LuaDist custom
 #
 # Version - read from 'version' file.
 #
@@ -735,3 +736,5 @@
 add_subdirectory(libarchive)
 add_subdirectory(tar)
 add_subdirectory(cpio)
+
+INSTALL(FILES README COPYING DESTINATION ${INSTALL_DATA}) # LuaDist custom
\ No newline at end of file
--- a/libarchive/CMakeLists.txt	2009-10-31 15:24:09.000000000 -0400
+++ b/libarchive/CMakeLists.txt	2009-11-18 01:19:01.781250000 -0500
@@ -131,10 +131,8 @@
 
 # How to install the libraries
 INSTALL(TARGETS archive archive_static
-        RUNTIME DESTINATION bin
-        LIBRARY DESTINATION lib
-        ARCHIVE DESTINATION lib)
+        DESTINATION ${INSTALL_LIB}) #LuaDist custom
 INSTALL_MAN(${libarchive_MANS})
-INSTALL(FILES ${include_HEADERS} DESTINATION include)
+INSTALL(FILES ${include_HEADERS} DESTINATION ${INSTALL_INC}) # LuaDist custom
 
 add_subdirectory(test)
